_data/stage3.yml

```
@@ -492,7 +492,7 @@
 - title: Promise.any
   id: proposal-promise-any
   has_specification: true
-  description: Introduce String.prototype.replaceAll as a way to replace all in a string
+  description: This proposal is to match popular usage in libraries, adds a method which returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected if all of the given promises are rejected.
```

the first clause (which needs a conjunction if kept) doesn't seem to add anything here to me.
```suggestion
  description: Adds a method which returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected if all of the given promises are rejected.
```

---

Agreed!